### PR TITLE
Partial linting fixes (same as #459 , with rebase and squash)

### PR DESCRIPTION
**Pull request by marcolincs**
### What are the issues this pull addresses (issue numbers / links)?
Same as #459 , but with rebase and squash of the commits to cleanup history, thus closes #459
Need to keep linting3 branch for future work.
Need to delete linting branch

Proof of equivalence (only 2 text changes, made on purpose while resolving merge conflicts during rebase)
image

### CODE REVIEW

**Comment by priti-ashvin-shah-ibm** [comment]
I had gone through manually all the changes in previous pr.  This time, I ran through the tutorial notebooks to keep a sanity check.  Looks as expected.

File: qiskit_metal/_gui/widgets/bases/QWidget_PlaceholderText.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
 # Any modifications or derivative works of this code must retain this
 # copyright notice, and modified files need to carry a notice indicating
 # that they have been altered from the originals.
-"""Ask Zlatko for help on this file"""
+"""Ask Zlatko for help on this file."""
```
Comment (priti-ashvin-shah-ibm) [inline qiskit_metal/_gui/widgets/bases/QWidget_PlaceholderText.py:14]: @zlatko-minev , Do you want this line to still be here?

Comment (zlatko-minev) [reply qiskit_metal/_gui/widgets/bases/QWidget_PlaceholderText.py:14]: Thanks, @priti-ashvin-shah-ibm . I think we can just change that at this point to """@author: Zlatko Minev""" to indicate the same

**Review by zlatko-minev — DISMISSED** [review]

**Review by priti-ashvin-shah-ibm — APPROVED** [review]
LGTM

**Pull request merged**